src/services/csv/eventTransform.js

```diff
@@ -186,6 +185,26 @@ module.exports = app => {
           actionRecipientAddress: pluginAddress,
           etherscanLink: getEtherscanLink(transactionHash),
         };
+        // eslint-disable-next-line no-restricted-syntax
+        for (const token of tokenWhiteList) {
```

Comment:
We should extract bridge info at the `addPayout` function which has donation entity and prevent another request to DB

Reply:
CSV service is slow already

Reply:
@aminlatifi 
It's a little complicated, I dont know how add it there , I think maybe I need your help for doing this (The complexity is that maybe we diffenet tokens and multiple donations in a row)